homeassistant/components/lutron_caseta/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "lutron_caseta",
   "name": "Lutron Cas\u00e9ta",
   "documentation": "https://www.home-assistant.io/integrations/lutron_caseta",
-  "requirements": ["pylutron-caseta==0.16.0"],
+  "requirements": ["pylutron-caseta==0.17.1"],
```

Comment:
```suggestion
  "requirements": ["pylutron-caseta==0.16.0"],
```

Reply:
not sure how that got in there... was working on that in a separate branch.  I've removed.